### PR TITLE
[RI01 CH06] Editorial changes in the Installer Prerequisite

### PR DESCRIPTION
**Pull request by EditKoselak**
closes: #1653

### CODE REVIEW

File: doc/ref_impl/cntt-ri/chapters/chapter06.md [hunk 1]
```diff
@@ -310,6 +310,6 @@ Before the installation, the user has to check if each server meets the deployme
 
 <a name="6.4.2"></a>
 ### 6.4.2 Network configuration
-It's very important to make sure the necessary prerequisite settings should be ready before the deployment.
-For example, user need configurate necessary vlan on the switch before deployment.
-And user need obtain the valid ip address planned from lab administrator.
+The necessary prerequisite settings must be ready before the deployment, for example:
+ - the VLAN must be configured on the switch,
+ - the used IP address ranges must be allocated.
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/chapters/chapter06.md:315]:
```suggestion
 - the IP address ranges to be used must be allocated.
```

**Comment by rabi-abdel** [comment]
@EditKoselak would it be possible please to address @pgoyal01 feedback above?

**Comment by rabi-abdel** [comment]
@EditKoselak would it be possible please to address @pgoyal01 feedback above?

File: doc/ref_impl/cntt-ri/chapters/chapter06.md (hunk 1 above)
Comment (EditKoselak) [reply doc/ref_impl/cntt-ri/chapters/chapter06.md]: Thanks, done:)

**Pull request merged**